ding/framework/middleware/learner.py

```diff
@@ -45,7 +45,7 @@ def __call__(self, ctx: "OnlineRLContext") -> None:
         Output of ctx:
             - train_output (:obj:`Deque`): The training output in deque.
         """
-        train_output_queue = deque()
+        train_output_queue = []
```

Comment:
this modification will conflict with [this](https://github.com/opendilab/DI-engine/blob/main/ding/framework/middleware/functional/data_processor.py#L129) implementation and its unitttest, please make them compatible.